src/main/resources/application.yaml

```diff
@@ -62,6 +62,7 @@ security:
     - "/favicon.ico"
   authorisedServices:
     - "wa_task_management_api"
+    - "ccd_gw"
```

Comment:
I'd expect we would need to allow ccd_something or other eventually but what made us make this change now? Is it part of the tests. Just interested but wondered if this got added and isn't really needed?